frontend/src/metabase/components/DateAllOptionsWidget/DateAllOptionsWidget.tsx

```diff
@@ -27,39 +26,41 @@ export const DateAllOptionsWidget = ({
   onClose,
   disableOperatorSelection,
   value,
+  defaultValue,
+  initialValue,
+  required = false,
 }: DateAllOptionsWidgetProps) => {
   const [filter, setFilter] = useState(
-    value != null ? dateParameterValueToMBQL(value, noopRef) || [] : [],
+    initialValue != null
```

Comment:
We need a separate `initialValue` to be used at first render because actual `value` is unset at this point, and it might break the button label selection logic.